community/joe/depends

```diff
@@ -0,0 +1,3 @@
+make
```

Comment:
make is an implicit dependency (ie, you don't need to include it in depends files).